kclvm/query/src/query.rs

```diff
@@ -106,30 +117,58 @@ pub fn get_schema_type(
     Ok(result)
 }
 
-fn resolve_file(file: &str, code: Option<&str>) -> Result<Rc<RefCell<Scope>>> {
+#[derive(Debug, Clone, Default)]
+pub struct CompilationOptions {
+    pub k_files: Vec<String>,
+    pub loader_opts: Option<LoadProgramOptions>,
+    pub resolve_opts: Options,
+    pub get_schema_opts: GetSchemaOption,
+}
+
+pub fn get_full_schema_type(
```

Comment:
Could U pls add more documents for public functions and structures?

Reply:
fixed

Reply:
fixed